### PR TITLE
Add multistre.am

### PR DESCRIPTION
**Pull request by nixxquality**

### CODE REVIEW

File: src/chrome/content/rules/multistre.am.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="multistre.am">
+	<target host="multistre.am" />
```
Comment (J0WI) [inline src/chrome/content/rules/multistre.am.xml]: Could you also add www.multistre.am?

Comment (nixxquality) [inline src/chrome/content/rules/multistre.am.xml]: There seems to be a bug with the website where https://www.multistre.am/ redirects to the HTTP version, which is why I didn't.
The www. only redirects to non-www anyway. Are you sure that I should?

Comment (J0WI) [inline src/chrome/content/rules/multistre.am.xml]: `https://www.multistre.am/` redirects to `http://multistre.am/` and we will redirect it to `https://multistre.am/`. That's totally fine.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**